src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -28,16 +30,123 @@ const ActorInit En_Part_InitVars = {
     (ActorFunc)EnPart_Draw,
 };
 
-#endif
+Vec3f D_80865900 = { 0.0f, 0.0f, 0.0f };
+
+static void (*D_8086590C[5])(EnPart*, PlayState*) = {
+    func_80865390, func_808654C4, 0x00000000, 0x00000000, 0x00000000,
+};
+
+void EnPart_Init(Actor* thisx, PlayState* play) {
+    EnPart* this = THIS;
+}
+
+void EnPart_Destroy(Actor* thisx, PlayState* play) {
+    EnPart* this = THIS;
+}
+
+void func_80865390(EnPart* this, PlayState* play) {
+    f32 randVar;
+
+    this->unk144 = 1;
+    this->actor.world.rot.y = (s16)(Rand_ZeroOne() * 20000.0f);
+    switch (this->actor.params) {
+        case 1:
+
+        case 4:
```

Comment:
We usually avoid leaving an empty line here
```suggestion
        case 1:
        case 4:
```